src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/XmlDataContract.cs

```diff
@@ -379,6 +380,7 @@ private bool RequiresMemberAccessForCreate(SecurityException securityException)
         }
 #endif
 
+        [RemovableFeature(ReflectionBasedSerializationFeature.Name)]
```

Comment:
Is it going to be an issue that this RemovableFeatureAttribute is not ifdef'ed for only uapaot?